### PR TITLE
Adding initial 508 Compliant MCP statements page

### PR DESCRIPTION
**Pull request by Dr-Pongo**
## Description
Creating new page with navigation for HTML Statements page

## Original issue(s)
department-of-veterans-affairs/va.gov-team#35871

## Acceptance criteria
- [x] Bare bones page and navigation is implemented

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by OptionSelect — APPROVED** [review]
LGTM! 🚀

File: src/applications/medical-copays/tests/e2e/statements.cypress.spec.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+/**
```
Comment (mdewey) [inline src/applications/medical-copays/tests/e2e/statements.cypress.spec.js:1]: Is this TestRail stuff still correct?

File: src/applications/medical-copays/containers/HTMLStatementPage.jsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+import scrollToTop from 'platform/utilities/ui/scrollToTop';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import Modals from '../components/Modals';
+import StatementAddresses from '../components/StatementAddresses';
+
+const HTMLStatementPage = ({ match }) => {
+  const selectedId = match.params.id;
+  const statements = useSelector(({ mcp }) => mcp.statements) ?? [];
+  const [selectedCopay] = statements?.filter(({ id }) => id === selectedId);
```
Comment (mdewey) [inline src/applications/medical-copays/containers/HTMLStatementPage.jsx:13]: So if statements are null, nothing on the page is going to be showing correctly and/or be useful? 

The question(s) here are 

- Do we care if `statements` is null/undefined and what does that mean for our page?
- Do we care if `selectedCopay` is null/undefined and what does that mean for the page?

`?.` can easily swallow errors, I tend to err on the side of `fail early and fail fast`. 

Nothing blocking this PR, but going forward we might want to look at better handling these checks. Whether is is showing an error page or always expecting these things to be there.

**Review by mdewey — APPROVED** [review]
Overall looks good, some things to consider but looks good

**Review by theodur — APPROVED** [review]

File: src/applications/medical-copays/containers/HTMLStatementPage.jsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+          className="vads-u-font-size--h3 vads-u-margin-top--0 vads-u-margin-bottom--5"
+          data-testid="facility-name"
+        >
+          {`${selectedCopay?.station.facilityName}`}
+        </p>
+        <Link
+          className="vads-u-font-size--sm"
+          to={`/balance-details/${selectedId}`}
+        >
+          <i
```
[the remark this answers is not in the document]
Comment (Dr-Pongo) [reply src/applications/medical-copays/containers/HTMLStatementPage.jsx:49]: Icons here are decorative and hidden with `aria-hidden="true"`

File: src/applications/medical-copays/containers/HTMLStatementPage.jsx [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+        <StatementAddresses id="statement-addresses" copay={selectedCopay} />
+        <h2>What if I have questions about my statement?</h2>
+        <Modals title="Notice of rights and responsibilities">
+          <Modals.Rights />
+        </Modals>
+        <Link
+          className="vads-u-font-size--sm"
+          to={`/balance-details/${selectedId}`}
+        >
+          <i
```
[the remark this answers is not in the document]
Comment (Dr-Pongo) [reply src/applications/medical-copays/containers/HTMLStatementPage.jsx:65]: Icons here are decorative and hidden with `aria-hidden="true"`

**Pull request merged**